package.json

```diff
@@ -35,15 +36,17 @@
     "storybook-native": "./node_modules/@storybook/react-native-server/bin/index.js -p 7007 -c ./storybook-native",
     "storybook-web": "./node_modules/@storybook/react/bin/index.js -c ./storybook-web -s ./public -p 5005 --ci",
     "upload-sourcemaps:release:android": "./scripts/upload-sourcemaps-release-android.sh",
-    "postinstall": "patch-package && jetify && cp ./node_modules/mapeo-offline-map/style.json ./android/app/src/main/assets/offline-style.json",
+    "postinstall": "patch-package && jetify && npm run build:intl-polyfills && cp ./node_modules/mapeo-offline-map/style.json ./android/app/src/main/assets/offline-style.json",
```

Comment:
wondering if i also need to update the `build:translations` script to call `npm run build:intl-polyfills`, because otherwise on initial install, the polyfills script wouldn't be working with the appropriate data?

Reply:
ah yeah just tried and because the `translations/` directory could be missing on initial install, my script doesn't work. maybe i'll move the call to this script to `build:translations` instead